### PR TITLE
Command to request pet status

### PR DESCRIPTION
**Pull request by paulo-romano**
I have finalized the implementation of the management command and admin action to request pet status. 

Now it's possible to request pet status of unsolved cases, just by a "python manage.py request_unsolvedcases_status" in terminal. And request status, of any type of case, by a action in  PetAdmin.

### CODE REVIEW

**Comment by dirtycoder** [comment]
Hi @paulo-romano, thanks for the PR.
Can you fix the broken test before I review this?

File: pets/meupet/models.py [hunk 1]
```diff
@@ -92,7 +103,7 @@ class Pet(models.Model):
                                         help_text='Tamanho máximo da imagem é 8MB')
     published = models.BooleanField(default=False)
     created = models.DateTimeField(auto_now_add=True)
-    modified = models.DateTimeField(auto_now=True)
+    modified = models.DateTimeField(auto_now_add=True)
```
Comment (dirtycoder) [inline pets/meupet/models.py]: I think `modified` will have the same value as the `created` attribute as both are using `auto_now_add`, take a look at the [docs](https://docs.djangoproject.com/en/1.9/ref/models/fields/#django.db.models.DateField.auto_now_add).

File: pets/meupet/services.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+from django.utils.timezone import timedelta, now
+
+
+def get_date_3_months_ago(target_date=now()):
+    return target_date - timedelta(days=90)
```
Comment (dirtycoder) [inline pets/meupet/services.py]: Blank line after this. :)

File: pets/meupet/tests/tests.py [hunk 3]
```diff
@@ -343,3 +364,23 @@ def test_get_pet_by_slug(self):
...
+        self.assertTrue(get_date_3_months_ago(today) == today_3months_ago)
+
+
+    def test_get_unsolved_cases_return(self):
+        """get_unsolved_cases method must return cases still open in 3 months"""
+        self.create_some_pets()
+
+        unsolved_cases = Pet.objects.get_unsolved_cases()
+
+        self.assertEqual(len(unsolved_cases), 2)
```
Comment (dirtycoder) [inline pets/meupet/tests/tests.py]: Blank line here.. :)

File: contrib/sample-env [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+DB_CONN_MAX_AGE=0
+SECRET_KEY=dummy_key
+ALLOWED_HOSTS=127.0.0.1, .localhost
+EMAIL_BACKEND=django.core.mail.backends.console.EmailBackend
+EMAIL_USE_TLS=False
+EMAIL_HOST=''
+EMAIL_PORT=465
+EMAIL_HOST_USER=''
+EMAIL_HOST_PASSWORD=''
+DEFAULT_FROM_EMAIL=''
```
Comment (dirtycoder) [inline contrib/sample-env]: Blank line here. :)

Comment (cuducos) [inline contrib/sample-env]: What about also adding a `SECRET_KEY` here?

**Comment by dirtycoder** [comment]
@paulo-romano could you submit the README as a separated PR?

**Comment by dirtycoder** [comment]
@paulo-romano as we talked about some time ago, I'll close this PR as I began to do it myself. :)
Thanks!

**Pull request closed without merging**